### ISSUE TITLE
Wireframe for ONRResource 508 compliance page

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1683]
@jkwonatx is going to do a wireframe, so Dave knows what to build.

I will send him screenshots of ONRResource and the list of documents to be organized.

**Comment by jkwonatx** [comment]
Shannon and I understood the problem. Here's the plan: 
1. understand the files 
2. categorize the files and create a file tree (contact Carrie for help)
3. compare the tree to how the files are currently organized in the intranet page 
4. low-fi wireframe sketches

**Comment by mcharg** [comment]
Link to miro board. https://miro.com/app/board/uXjVOZY8bT8=/?invite_link_id=923924300917

**Comment by jkwonatx** [comment]
onrresources files excel spreadsheet

[https://app.zenhub.com/files/210892300/9dfde446-b101-4019-820a-c4cc73536c5c/download](https://app.zenhub.com/files/210892300/9dfde446-b101-4019-820a-c4cc73536c5c/download)

**Comment by jkwonatx** [comment]
[https://app.zenhub.com/files/210892300/55d7e0a1-9492-4672-8ac7-486de10c3eac/download]
(ONRR 508 files recategorized)

- [ ] Delete unnecessary or duplicate resource items
- [ ] Add missing items
- [ ] Rename the files as necessary
- [ ] Rename the category names as necessary

**Comment by mcharg** [comment]
@jkwonatx Here's the updated version where @cmathers1 and I got rid of duplicates, put everything into categories and subcategories, and logically ordered everything within each category.  I didn't see your note about renaming files until after we finished our meeting, so I'll take a look at that in the morning.

https://doimspp.sharepoint.com/:x:/r/sites/NRRD/Shared%20Documents/General/ONRR%20508%20Compliance%20Approach%20Team/ONRResources.508.Compliance.xlsx?d=wec83f76fa29249789220c3cd5b92cb2e&csf=1&web=1&e=hjCU6F

**Comment by jkwonatx** [comment]
![Screen Shot 2022-04-04 at 8 15 32 PM](https://user-images.githubusercontent.com/85380376/161658847-38492021-3f37-419c-9b04-674737c586b6.png)

**Comment by jkwonatx** [comment]
What we've accomplished: 
![Screen Shot 2022-04-04 at 8 18 16 PM](https://user-images.githubusercontent.com/85380376/161659082-f7bdd327-4f8b-434f-9d09-e4b04c315a0e.png)
Where we are in the process:
![Screen Shot 2022-04-04 at 8 17 47 PM](https://user-images.githubusercontent.com/85380376/161659088-969911c8-4b50-44ff-8493-2939d9785dce.png)
Next -- fat marker sketches, answer wireframe questions (use Axure? design from scratch or use ONRR design elements? - @mcharg?)

**Comment by mcharg** [comment]
@jkwonatx You can use Axure or whatever you want to use. I would use elements that are on the ONRResource pages in the screenshots.

**Comment by mcharg** [comment]
@jkwonatx @cmathers1 Titles are updated in the spreadsheet.
https://doimspp.sharepoint.com/:x:/r/sites/NRRD/Shared%20Documents/General/ONRR%20508%20Compliance%20Approach%20Team/ONRResources.508.Compliance.xlsx?d=wec83f76fa29249789220c3cd5b92cb2e&csf=1&web=1&e=5e7QGu

**Comment by mcharg** [comment]
Next step is #1873 for a clickable prototype.
